pokemongo_bot/__init__.py

```diff
@@ -212,6 +212,8 @@ def work_on_cell(self, cell, position):
                     if hack_chain > 10:
                         #print('need a rest')
                         break
+                    if self.config.mode == "poke":
```

Comment:
This is a reasonable fix for now, but probably not the right long term fix. We should be removing this loop entirely so it instead only acts on one fort per event loop.